src/main/java/frc/robot/subsystems/ControlPanelSubsystem.java

```diff
@@ -47,18 +46,18 @@ public void stopMotor() {
   public void engageControlPanel() {
     pistonsForward = !pistonsForward;
     if (pistonsForward) {
-      // controlPanelSolenoid.set(Value.kForward);
+      controlPanelSolenoid.set(Value.kForward);
```

Comment:
So we don't duplicate this code and it's easier to think about what's happening.

```suggestion
      wheelUp();
```

We also might want to change this method name from `engageControlPanel()` to `toggleControlPanel()`.